index.html

```diff
@@ -11,7 +11,7 @@
             border: 10px solid skyblue;
         }
         p{
-            color: white;
+            color: greenyellow;
```

Comment:
But now the border and the text are the same color! This will be very feo.

Reply:
Pero no te pongas un like a tu comentario.. has de ponerlo a la petición xD